client/src/components/SharedComponents/Signup/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import './signup.css'
+
+class Signup extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    confirm_password: '',
+    displayBio: false
+  }
+
+  onChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  ShowDisplayBio = () => {
+    this.setState({ displayBio: true })
+  }
+
+  lessDisplayBio = () => {
+    this.setState({ displayBio: false })
+  }
+
+  validateForm = () => {
+    return (
+      this.state.name.length > 2 &&
+      this.state.email.length > 6 &&
+      this.state.password.length > 3 &&
```

Comment:
password length of 4 is still very easy to crack through brute force attacks. If you wanna validate password, it should be 6+ characters. That's when hashing makes it near impossible to guess.
[This website](https://www.betterbuys.com/estimating-password-cracking-times/) gives u a really fun tool to play around with passwords and see how difficult it is to crack.